src/components/AppNavigation/AppointmentConfigList/AppointmentConfigListItem.vue

```diff
@@ -24,6 +24,7 @@
 	<div>
 		<AppNavigationItem
 			:title="config.name">
+			<CalendarCheckIcon slot="icon" :size="20" decorative />
```

Comment:
```suggestion
			<template #icon>
				<CalendarCheckIcon :size="20" decorative />
			</template>
```

Reply:
yes, sorry :(, thank you for carefully checking it

Reply:
@raimund-schluessler I'm curious about this change. AFAIK, the original version (`slot=""` attribute) is working fine too. Could you explain the reasons for using `<template>` vs `slot=""`.

Reply:
The slot attribute is deprecated and got removed in vue3. Although it will still work for all 2.x releases of vue, we should still use the new syntax. See https://vuejs.org/v2/guide/components-slots.html#Deprecated-Syntax

Reply:
TIL! Thanks for sharing.